snuba/datasets/entities/events.py

```diff
@@ -71,17 +66,18 @@ class EventsEntity(Entity):
     and the particular quirks of storing and querying them.
     """
 
-    def __init__(self) -> None:
+    def __init__(self, custom_mappers: Optional[TranslationMappers] = None) -> None:
```

Comment:
I think this is an issue and sorry for missing it in the transactions entity. Allowing parameters in the constructor of the entity (which is something built by the factory) would open the door to instantiating an events entity with any sort of translators, which would not be an events entity anymore (the fact that the factory ensures this does not happen is an optimization as this is a valid constructor and thus could be used). I think we had similar discussions months ago around initialization parameters for storages or daasets.

I think leaving this open is unnecessary and it would be easy to close by having an abstract events entity that takes these parameters and cannot be instantiated (being abstract) and making two concrete entities EeventsEntity and EventsDiscoverEntity that do not take constructor parameters. 

Since this is already done for transactions feel free to do it as a follow up

Reply:
I can tackle this as a follow up - please keep in mind this is just temporary stuff and taking the constructor parameter won't exist once we are rid of the second temporary entity.

Reply:
ok